### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by joeliths**

### CODE REVIEW

**Comment by DaniellaNoren** [comment]
Comment

File: ReadMe [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Test
+This
```
Comment (DaniellaNoren) [inline ReadMe:2]: dfgdfgdfg

File: ReadMe [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Test
```
Comment (DaniellaNoren) [inline ReadMe:1]: dfgdgdfg

File: ReadMe (hunk 1 above)
Comment (IdaFagerlund) [inline ReadMe:2]: That

File: ReadMe [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+This
+Is
+a
```
Comment (IdaFagerlund) [inline ReadMe:4]: an

File: ReadMe (hunk 1 above)
Comment (DaniellaNoren) [reply ReadMe:2]: whaaat

**Review by IdaFagerlund — CHANGES_REQUESTED** [review]
Great!

**Review by DaniellaNoren — CHANGES_REQUESTED** [review]
bad

**Pull request merged**